javascript/ql/test/library-tests/ModelGeneration/ModelGeneration.ext.yml

```
@@ -0,0 +1,7 @@
+extensions:
+  - addsTo:
+      pack: codeql/javascript-all
+      extensible: typeModel
+    data:
+      - ["upstream-lib.XYZ", "upstream-lib", "Member[x].Member[y].Member[z]"]
+      - ["upstream-lib.Type", "upstream-lib", "Member[Type].Instance"]
```

Maybe it's my JS knowledge lacking, but I would have thought this should be

```suggestion
      - ["upstream-lib.Type", "upstream-lib", "Member[Type]"]
```

I can see that when you fixed this problem, you did not change the typeModel in the .ext.yml file, so I'm probably wrong... but that's just an opportunity to learn :blush:

---

`upstream-lib.Type` means "an instance of the type `Type` from `upstream-lib`".

See [reference on types in JS MaD](https://codeql.github.com/docs/codeql-language-guides/customizing-library-models-for-javascript/#types)